cfme/tests/containers/test_pause_resume_workers.py

```diff
@@ -0,0 +1,69 @@
+from cfme.containers.provider import ContainersProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+import pytest
```

Comment:
Imports need to be sorted, pytest is a pip installable package which should be above the local imports, with an empty line between them.  More details in the docs.